tests/test_configurator.py

```diff
@@ -1,98 +1,162 @@
 # Ensure Python 2/3 compatibility: http://python-future.org/compatible_idioms.html
 from __future__ import print_function
 from imp import reload
+from future.utils import iteritems
 import six
 
+import mock
+from pyfakefs import fake_filesystem
+import nomenclate.core.configurator as config
 import unittest
 import mock
-import configparser
+import collections
 from pyfakefs import fake_filesystem
 import nomenclate.core.configurator as config
 from collections import OrderedDict
 
-reload(config)
 
-__author__ = "Andres Weber"
-__email__ = "andresmweber@gmail.com"
-
-@unittest.skip("skipping until finished testing nomenclate")
+#@unittest.skip("skipping until finished testing nomenclate")
```

Comment:
So if there's a comment like this, how can we close it?  Or should we keep it just as a log?